scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -22,48 +22,109 @@ import mill.scalalib.JavaModule
 trait AndroidAppModule extends JavaModule {
 
   /**
-   * Abstract method to provide access to the Android SDK configuration.
-   *
-   * This method must be implemented by the concrete class to specify the SDK paths.
-   *
-   * @return The Android SDK module that is used across the project.
+   * Provides access to the Android SDK configuration.
    */
   def androidSdkModule: ModuleRef[AndroidSdkModule]
 
   /**
-   * An XML file containing configuration and metadata about your android application
+   * Loads `AndroidManifest.xml` file.
    */
   def androidManifest: Task[PathRef] = Task.Source(millSourcePath / "AndroidManifest.xml")
 
   /**
-   * Generates the Android resources (such as layouts, strings, and other assets) needed
-   * for the application.
+   * Adds "aar" to the handled artifact types.
+   */
+  def artifactTypes: T[Set[coursier.Type]] = Task { super.artifactTypes() + coursier.Type("aar") }
```

Comment:
there is `aar` support added here, but examples lack any 3rd party dependency, is it intentional?

Reply:
As mentioned below this will be used in future in jetpack compose or third-party addition examples